aladdin.sh

```diff
@@ -378,4 +386,5 @@ copy_ssh_to_minikube
 set_cluster_helper_vars
 handle_ostypes
 prepare_docker_subcommands
+synchronize_datetime
```

Comment:
I'm a bit worried on performance, do we need to run this every time?

Reply:
can we detect time skew and then run this based on that?

Reply:
I think we can detect time skew by putting the host time on an env variable when starting the container, then check the diff between that and the aladdin container time, if that is greater than X threshold then run the synchronize

Reply:
How concerned are we about performance?  I ran this 10 times and it looked like it was averaging about 3/4 of a second.  Is that worth optimizing to avoid?  In particular, I don't imagine we can detect clock skew without sshing into minikube, which is what takes the majority of that time (just setting the date from inside the minikube vm takes 0.001s)

Reply:
no love for the "do on init" suggestion?